client/src/locales/en-US.json

```diff
@@ -348,7 +348,7 @@
     },
     "WorkspacesPage": {
         "createWorkspace": "New workspace",
-        "itemCount": "No items | 1 item | {count} items",
+        "itemCount": "Items : 0 | Item : 1 | Items: {count}",
```

Comment:
```suggestion
        "itemCount": "Items: 0 | Item: 1 | Items: {count}",
```

Reply:
Why change `No items` here but not after ? I'd say `No items` is better than `Items: 0`

Reply:
I also prefer `No items`